project/Libs.scala

```diff
@@ -36,6 +36,7 @@ object Libs {
   val `HdrHistogram`      = "org.hdrhistogram"         % "HdrHistogram"      % "2.1.12"
   val `testng`            = "org.testng"               % "testng"            % "7.3.0"
   val `junit-4-12`        = "org.scalatestplus"       %% "junit-4-12"        % "3.2.2.0"
+  val `junit-4-13`        = "org.scalatestplus"       %% "junit-4-13"        % "3.2.5.0"
```

Comment:
As you probably saw, i needed 4.13 to use Assert.assertThrows.  So i made ParamsJVM dependent on this version.  But I didn't want to move all CSW projects to 4.12 because it seemed to me like this would be a big change and I didn't know if it would cause any issues.  If you think it's not an issue, we could get rid of 4.12.

Reply:
Yes, I pushed updated version. `SupervisorLifecycleFailureTest` test failed which is not related to this and is flaky.
Merging this PR.